divide_and_conquer/fibonacci_sequence_DNC.py

```diff
@@ -0,0 +1,50 @@
+"""
+We can find the Fibonacci sequence more effectively than naive bottom-up algorithm
+
+	 We use One of the well-known properties of Fibonacci number is:
+
+	 | F(n+1) ,  F(n)   |       | 1 , 1 | ^ n
+	 |                  |   =   |       |
+	 | F(n)   ,  F(n-1) |       | 1 , 0 |
+
+	 We can therefore compute the (n)th power of A = ((1,1),(1,0)) using recursive squaring
+
+	 Time complexity: O(log(n)), because it is a divide and conquer algorithm
+"""
+
+# return (n)th term of Fibonacci sequence
+def getFibonacci(n):
```

Comment:
Please have a look at CONTRIBUTING.md
We expect `method` and `variable` names to be in **snake_case**

Reply:
+ type hints, doctests and https://travis-ci.com/TheAlgorithms/Python/builds/140784236